openwisp_ipam/apps.py

```diff
@@ -1,11 +1,19 @@
-from django.apps import AppConfig
+from django.conf import settings
+from openwisp_utils.api.apps import ApiAppConfig
+from openwisp_utils.utils import default_or_test
 
 from .compat import patch_ipaddress_lib
 
 
-class OpenWispIpamConfig(AppConfig):
+class OpenWispIpamConfig(ApiAppConfig):
     name = 'openwisp_ipam'
     verbose_name = 'IPAM'
 
-    def ready(self):
+    API_ENABLED = getattr(settings, 'OPENWISP_IPAM_API', True)
```

Comment:
this should be probably always true in this module

Reply:
Yeah, I'll do it in the next commit.